cpp/cmake/thirdparty/get_cuvs.cmake

```diff
@@ -73,8 +73,8 @@ endfunction()
 # To use a different CUVS locally, set the CMake variable
 # CPM_cuvs_SOURCE=/path/to/local/cuvs
 find_and_configure_cuvs(VERSION          ${CUML_MIN_VERSION_cuvs}
-      FORK             rapidsai
-      PINNED_TAG       branch-${CUML_BRANCH_VERSION_cuvs}
+      FORK             divyegala
```

Comment:
Just noting that we'll need to make sure to switch these back once the associated PRs have been merged so we don't forget.